plugins/woocommerce/client/admin/client/core-profiler/pages/Plugins/components/plugin-terms-of-service/test/index.tsx

```diff
@@ -17,8 +17,8 @@ describe( 'PluginsTermsOfService', () => {
 			<PluginsTermsOfService
 				selectedPlugins={ [
 					{
-						key: 'woocommerce-payments',
-						name: 'WooCommerce Payments',
+						key: 'test-plugin',
+						name: 'test-plugin',
```

Comment:
@moon0326 Was the `test-plugin` merge intentional? 🤔

Reply:
Yes! The test was failing since we're rendering TOS for woocommerce-payments now.

Reply:
@moon0326  Nice, thanks for catching it!